samples/bootloader/README.rst

```diff
@@ -0,0 +1,170 @@
+.. _bootloader:
+
+Immutable bootloader
+####################
+
+The bootloader sample implements an immutable first stage bootloader that has the capability to verify and boot a second stage bootloader.
+If the second stage bootloader is upgradable, this sample also provides a mechanism to upgrade the second stage bootloader.
```

Comment:
Do we have a second stage bootloader that we could point to? Or just point to mcuboot maybe?

Reply:
MCUBoot is an example of (the only one currently supported) a second stage bootloader.